src/client/components/componentWithSliders/allSliders/allSliders.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './allSliders.css';
+
+import BlockOfSliders from './blockOfSliders';
+
+const BLOCKS = [
+  { name: 'Distortion', numOfSliders: 2 },
+  { name: 'Flenger', numOfSliders: 4 },
+  { name: 'Reverb', numOfSliders: 2 },
+  { name: 'Delay', numOfSliders: 3 },
+  { name: 'Tremble', numOfSliders: 1 }
+];
+
+const AllSliders = () => {
+  const blocks = [];
+  for (let i = 0; i < BLOCKS.length; i++) {
```

Comment:
there is a better way to render elements in a loop in react, you can refer to this docs https://reactjs.org/docs/lists-and-keys.html
Better will be to map through your elements and return an array of jsx
If more info recruited, contact me

Reply:
fixed